src/niscope/metadata/functions_addon.py

```diff
@@ -2,6 +2,41 @@
 # Changes to the metadata should be made here, because functions.py is generated thus any changes get overwritten.
 
 functions_override_metadata = {
+    'ClearWaveformMeasurementStats': {
```

Comment:
The idea with leaving the `*_addon.py` files was to fix issues quickly when we couldn't get a change through the internal metadata and exported for here.

Anything here should generally have an internal change to metadata that matched. Then add a comment along the lines that this can be removed from here once the metadata is updated. The only things that would be in a `*_addon.py` file but _not_ in the internal metadata is stuff that only applies to GitHub such as `'module_version'` in config.

Reply:
Addressed in #1492